app/views/materials/compare.html.erb

```diff
@@ -1,19 +1,17 @@
 <div class="jumbotron">
   <form>
     <% @materials.each_with_index do |material, index| %>
-    <% value = params["material_#{index}_uuid"] %>
       <div class="row">
         <div class="col mb-4">
-          <%= label_tag "material_#{index+1}_uuid" %>
-          <%= text_field nil, "material_#{index}_uuid", value: value, class: "form-control", placeholder: "UUID" %>
+          <%= label_tag "material #{index}" %>
+          <%= text_field nil, "material_ids[]", value: params[:material_ids][index], class: "form-control", placeholder: "UUID" %>
```

Comment:
Could use `material.id` for the value no?

Reply:
The material might not be present.